CONTRIBUTING.md

```diff
@@ -70,16 +88,17 @@ Our code base is written in TypeScript and must adhere to specific conventions a
 
 ## Getting a development environment set up
 
-We recommend installing the following extensions in your editor of choice: TypeScript, Sass, TailwindCSS, ESLint and Prettier.
+An installation of Node is required for development. If you don't have Node installed, we recommend [Volta], which will automatically use the Node versions we pinned in `package.json`. We also recommend installing the following extensions in your editor of choice: TypeScript, TailwindCSS, ESLint, StyleLint, and Prettier. If you use VS Code, you will see a pop up in the bottom right corner prompting you to install or view the workspaces's recommended extensions. You can install the extensions manually if you do not use VS Code:
 
-- https://code.visualstudio.com/
+- https://tailwindcss.com/docs/intellisense
+- https://eslint.org/docs/latest/user-guide/integrations
+- https://stylelint.io/user-guide/integrations/editor
 - https://atom.io/packages/atom-typescript
 - https://github.com/Microsoft/TypeScript-Sublime-Plugin
```

Comment:
Should these two extensions be mentioned if we're referencing no specific editor above?

Reply:
Thanks for all the feedback! I'll make the changes. Yeah I was on the fence about this too but opted for keeping them since they were already there. I'll remove them.